stdlib/SparseArrays/test/sparse.jl

```diff
@@ -2855,4 +2855,14 @@ end
     @test SparseArrays.getcolptr(vA) == SparseArrays.getcolptr(A[:, 1:5])
 end
 
+@testset "mapreducecols" begin
+    n = 20
+    m = 10
+    A = sprand(n, m, 0.2)
+    B = mapreduce(identity, +, A, dims=2)
```

Comment:
I'd suggest to also test against the full matrix case. Like, add here:
```julia
@test B ≈ mapreduce(identity, +, Matrix(A), dims=2)
```
and then merge.